keras_nlp/models/bert/bert_tasks.py

```diff
@@ -123,44 +204,70 @@ def from_preset(
     ):
         """Create a classification model from a preset architecture and weights.
 
+        By default, this method will automatically create a `preprocessor`
+        layer to preprocess raw inputs during `fit()`, `predict()`, and
+        `evaluate()`. If you would like to disable this behavior, pass
+        `preprocessor=None`.
+
         Args:
             preset: string. Must be one of {{names}}.
             load_weights: Whether to load pre-trained weights into model.
                 Defaults to `True`.
 
         Examples:
+
+        Raw string inputs.
         ```python
-        input_data = {
+        # Create a dataset with raw string features in an `(x, y)` format.
+        features = ["The quick brown fox jumped.", "I forgot my homework."]
+        labels = [0, 3]
+        ds = tf.data.Dataset.from_tensor_slices((features, labels)).batch(2)
```

Comment:
Do we need `Dataset` conversions in the docstring? Seems kinda heavy

Reply:
Updated this to stop showing tf datasets, and just show passing in tensor/python inputs.